plugins/lookup/revbitspss.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, RevBits <info@revbits.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+name: revbitspss
+author: RevBits (@info) <info@revbits.com>
+short_description: Get secrets from RevBits PAM server
+version_added: 3.8.0
+description:
+    - Uses the revbits-ansible Python SDK to get Secrets from RevBits PAM
+      Server using API key authentication with the REST API.
+requirements:
+    - revbits-ansible - U(https://pypi.org/project/revbits-ansible/)
+options:
+    _terms:
+        description:
+            - This will be an array. First index will conatin another array of keys for secrets which you want to fetch from RevBits PAM.
+            - At second index you need to pass the base URL of the server, for example C(https://pam.revbits.net).
+            - At third index you need to place API key for auth. You can get from RevBits PAM secret manger module
+        type: list
+        elements: str
+        ini:
+            - section: revbitspss_lookup
+              key: _terms
+        required: true
+"""
+
+RETURN = r"""
+_list:
+    description:
+        - The JSON responses which you can access with defined keys.
+        - If you are fetching secrets named as UUID, PASSWORD it will gives you the dict of all secrets.
+    type: list
+    elements: dict
+"""
+
+EXAMPLES = r"""
+- hosts: localhost
+  vars:
+      secret: >-
+        {{
+            lookup(
+                'community.general.revbitspss',
+                 ['UUIDPAM','DB_PASS','DB_USER'],
+                 'https://pam.revbits.net',
+                 'API_KEY_GOES_HERE'
+            )
+        }}
+
+  tasks:
+      - ansible.builtin.debug:
+          msg: >
+            UUIDPAM is {{ (secret['UUIDPAM']) }} and DB_PASS is {{ (secret['DB_PASS']) }} and DB_USER is {{ (secret['DB_USER']) }}
+"""
+
+from ansible.errors import AnsibleError, AnsibleOptionsError
+
+sdk_is_missing = False
+try:
+    from pam.revbits_ansible.server import (
+      SecretServer,
+      SecretServerError,
+    )
+except ImportError:
+    sdk_is_missing = True
```

Comment:
You should catch the exception and use the method described in https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/testing/sanity/import.rst to make the exception available to the user.